### DISCUSSION TITLE
Manual upgrade documentation instructions

### DISCUSSION THREAD

**Discussion by j-kwan** [babybuddy/babybuddy#875] [Support] [answered]
Hello
I followed the manual instructions to install version 2.5.0 on a virtual machine running proxmox. 
Are there specific instructions for performing the upgrade? 
Or do I just have to do a 'git pull' from '/var/www/babybuddy/public' and then restart the 'uwsgi' service?
Best regards,

**Comment by cdubz** [comment, marked as answer]
@j-kwan the one additional thing you may do to is run migrations. If you followed the manual deploy steps, you'll need to make sure the directory containing the database and the database itself are writable by your use user and run:

```shell
export DJANGO_SETTINGS_MODULE=babybuddy.settings.production
python manage.py migrate
```

After migrations complete, re-secure the the directory and database per the manual deployment steps.
